packages/dev/loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -205,10 +206,13 @@ export class GLTFLoader implements IGLTFLoader {
     private _uniqueRootUrl: Nullable<string> = null;
     private _gltf: IGLTF;
     private _bin: Nullable<IDataBuffer> = null;
-    private _babylonScene: Scene;
+    /** @internal */
+    public _babylonScene: Scene;
     private _rootBabylonMesh: Nullable<Mesh> = null;
     private _defaultBabylonMaterialData: { [drawMode: number]: Material } = {};
     private readonly _postSceneLoadActions = new Array<() => void>();
+    /** @internal */
+    public _pathToNodesMapping: { [key: string]: Node } = {};

```

Comment:
We discussed this offline also. There is no need for this mapping. We can create a cache to improve performance, but the JSON pointer should allow the code to reach the correct object under the `_gltf` tree which has internal fields for what Babylon runtime objects have been created for the glTF JSON object.